src/Resolvers/Microsoft.NET.Sdk.WorkloadManifestReader/SdkDirectoryWorkloadManifestProvider.cs

```diff
@@ -142,6 +143,13 @@ bool TryGetWorkloadSet(string workloadSetVersion, out WorkloadSet? workloadSet)
                     }
                 }
 
+                // The baseline workload sets were merged with a fixed 8.0.100 feature band. That means they will always be here
+                // regardless of where they would otherwise belong. This is a workaround for that.
+                if (workloadSets80100.TryGetValue(workloadSetVersion, out workloadSet))
```

Comment:
won't the baseline workload set get moved to 9.0.100 so this is only a temporary workaround? Is there no solution that doesn't involve us later needing to remove this?

Reply:
Yes, and it was actually already there (see lines 136-144). But it wasn't working because it assumed that you would have your workload sets associated with the feature band of that workload set, which wasn't true for these baseline workload sets.

So this workaround works for now, and once we actually put the baselines in the right folder, they'll get picked up by the other code.